profiles/factory.py

```diff
@@ -28,6 +29,8 @@ def create_app(config: Config, clients: Clients) -> Flask:
     orcid_tokens = SQLAlchemyOrcidTokens(db)
     profiles = SQLAlchemyProfiles(db)
 
+    uri_signer = URLSafeSerializer(config.orcid['webhook_key'])
```

Comment:
it seems the default in our version is SHA-1, but `master` that uses SHA-512 does not have a release yet:
https://github.com/pallets/itsdangerous/blob/0.24/itsdangerous.py#L297
https://github.com/pallets/itsdangerous/blob/master/itsdangerous.py#L308
We should specify SHA-512 ourselves not to rely on a default that is going to change.

Also `key_derivation` should be `hmac`, not the Django default https://github.com/pallets/itsdangerous/blob/0.24/itsdangerous.py#L307